mlir/python/mlir/dialects/linalg/__init__.py

```diff
@@ -102,3 +104,45 @@ def broadcast(
     )
     fill_builtin_region(op.operation)
     return op
+
+
+@register_attribute_builder("IteratorTypeArrayAttr")
+def _IteratorTypeArrayAttr(x, context):
+    return ArrayAttr.get([_iteratortypeenum(v, context) for v in x])
+
+
+class GenericOp_(GenericOp):
```

Comment:
need the underscore here so that opdsl generation doesn't break...

Reply:
Can you add a comment for future reference?